### DISCUSSION TITLE
Missing Versions

### DISCUSSION THREAD

**Discussion by davidlas** [nasko222/the-minecraft-recreation-project#153] [General]
Since I've been downloading the versions, I noticed that you don't have 0.0.14a_04 and 0.0.14a_05 reuploads as well as 0.0.15a_03 at all. The changes are questionable, but I'd like to let you know.

**Comment by nasko222** [comment]
The project doesn't recreate all ALL versions. Actually the repo used to have all of the ones you listed, including reuploads of 16a_02, 19a_06, and 21a. But they got removed because they're the same except they have a bug like "You cant play in SP, You cant play in MP" and so on. If you really want them (they are very unnecessary) you can revert the repo to older point (Try not to fork it.) and download them.

**Reply by G23M08** [reply]
How do I do that
Edit: yay

**Reply by nasko222** [reply]
good catch, im gonna add those to the MORE VERSIONS discussion later
